### ISSUE TITLE
No 1.21

### ISSUE THREAD

**Issue by lzm11111111** [Leximon/Fluidlogged#32]

**Comment by OnyxXIII** [comment]
Was this resolved, but just failed to push to mod sites?

**Comment by OnyxXIII** [comment]
@Leximon @ExpensiveKoala I hate to tag, but I just want to check in again. The server I admin was using Towelette for this, but it's not been updated in some time, and we're looking to move to 1.21.
We really like the work y'all have been doing here, and I just want to make sure y'all caught the error on the update to 1.21 process here

![Image](https://github.com/user-attachments/assets/f147c5f4-09e3-4459-a20e-c91f106bf761)

**Comment by ExpensiveKoala** [comment]
That error is unrelated to 1.21 port. It is the CI/CD build failing because it is using Java 17 instead of Java 21+.
https://github.com/Leximon/Fluidlogged/blob/6d7df7d0e3b65d6341c25ffa3ade6b9aede84d07/.github/workflows/build.yml#L9

The 1.21 branch works otherwise.

There were talks of addressing https://github.com/Leximon/Fluidlogged/issues/29 before releasing 1.21. I've been too busy to help with that for now.

**Comment by OnyxXIII** [comment]
oh, cool! I saw the merge for "Update to 1.21" and I wasn't sure if that was a completed update that just wasn't rolled out or if it was just part of the process.

I'll freely admit I have no idea how much work it takes to make an update like this, and I definitely don't want to be putting undue pressure on either of you.
I can say I'm looking forward to the update, though, if that helps 😄

**Comment by ExpensiveKoala** [comment]
Yeah, right now the 1.21 branch has a straight port. If you're just using it for your server with friends, you could probably just build from source and use it. If you're waiting on an official release, it'll be a bit longer.

**Comment by OnyxXIII** [comment]
Got docs on building from source? I haven't done any MC modding, but I'm decently comfortable with code so I could do it if I just had some documentation to reference

**Comment by ExpensiveKoala** [comment]
Just clone the repo, run `gradlew build` and you'll find the jar in `<neoforge or fabric>\build\libs`

**Comment by OnyxXIII** [comment]
Fantastic, thanks! I'll try that!

**Comment by OnyxXIII** [comment]
Works! Fantastic, thank you!

**Comment by 7OOTnegaTerces** [comment]
> Yeah, right now the 1.21 branch has a straight port. If you're just using it for your server with friends, you could probably just build from source and use it. If you're waiting on an official release, it'll be a bit longer.

Any update on the progress for an official release for 1.21?  It looks like there will be a NeoForge version!
